tests/console/udisks2.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: 
+# Maintainer: João Walter Bruno Filho <bfilho@suse.com> 
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # install udisks2 package
+    zypper_call('in udisks2');
+
+    # Check disk status 
+    validate_script_output("udisksctl status", sub { m/Disk/s });
+
+    # Check if Udisks2 output is valid (contains a sections for Block and Partition Table).
+    validate_script_output("udisksctl info --block-device /dev/vda", sub { m/UDisks2.*Block.*PartitionTable/s} );
+
+    # create or dowload iso file and map it by loop-setup
+    assert_script_run("wget https://download.opensuse.org/distribution/leap/15.0/iso/openSUSE-Leap-15.0-NET-x86_64.iso");
+    validate_script_output("udisksctl loop-setup -r -f openSUSE-Leap-15.0-NET-x86_64.iso", sub { m/Mapped/s });
```

Comment:
I think would be better to create small iso locally  and don't download it. Could you please try it?

Reply:
So, instead of downloading, I changed it to creating a local directory and packing it into iso and then mounting it with udisksctl. Would you please check this approach in the latest changes?